tendrl/monitoring_integration/grafana/dashboard.py

```diff
@@ -30,15 +30,21 @@ def upload_default_dashboards():
     datasource.create()
     dashboards = dashboard_utils.get_all_dashboards()
     for dashboard_json in dashboards:
-        title.append(dashboard_json["uri"].split('/')[1])
-
-    for dashboard_json in NS.config.data["dashboards"]:
-        if dashboard_json in title:
```

Comment:
This if condition is checking two completely different things, but its not failing as we are going down straight to creating the dashboard again. The dashboards cannot get updated as the grafana rejects the request with : {u'status': u'name-exists', u'message': u'A dashboard with the same name already exists'} .